Add bpf-developer-tutorial

yunwei37 opened the pull request:
Add the tutorial repo https://github.com/eunomia-bpf/bpf-developer-tutorial

qmonnet reviewed (commented):
The tutorials look great, nice work!

Would it make more sense to add https://eunomia.dev/tutorials/ instead of the GitHub repo? It seems that it's easier to read (and the GH repo is linked from there)? Or is that page outdated (I see the XDP tutorial is missing on the English version, at least)? Let me know, I'm fine with merging in the current state, too.

A suggestion for the README on the GH repo: I would maybe highlight the link to the English version more (bigger font / bold text / emojis...?), I spent maybe twenty seconds looking for the link the first time I landed on the page.

yunwei37 commented:
Thank you very much for reviewing the PR and providing your valuable feedback! I'm grateful for your suggestion to include the https://eunomia.dev/tutorials/ link rather than the GitHub repo.

Regarding the web pages, they are automatically deployed through the CI in the GitHub repo. I apologize for any link errors that may have occurred due to recent deployments(Because we have just set up the webpages a few weeks ago...). I've addressed these issues and fixed the links. Additionally, I've made sure to highlight the link to the English version more prominently in the README.

Your insights have been extremely helpful in improving the content.

qmonnet reviewed (approved):
No need to apologise :sweat_smile:. I'm glad I could be helpful. Looks all good from my side, thank you!
